modules/node_modules/@ncigdc/components/ProjectVisualizations.js

```diff
@@ -49,24 +48,25 @@ const enhance = compose(
   withState('selectedFrequentMutationsSurvivalData', 'setSelectedFrequentMutationsSurvivalData', {}),
   withState('state', 'setState', initialState),
   withState('numCasesAggByProject', 'setNumCasesAggByProject', undefined),
-  withPropsOnChange(['projectId', 'numCasesAggByProject', 'survivalPlotSampleSize'], async ({ projectId, setState, numCasesAggByProject, survivalPlotSampleSize }) => {
-    if (!numCasesAggByProject) {
-      return;
+  withPropsOnChange(
+    ['projectId', 'caseCount', 'survivalPlotSampleSize'],
+    async ({ projectId, setState, caseCount, survivalPlotSampleSize }) => {
+      if (!projectId) return;
+
+      const defaultSurvivalData = await getDefaultCurve({
+        totalCases: caseCount,
+        currentFilters: { op: '=', content: { field: 'cases.project.project_id', value: projectId } },
```

Comment:
Ok I think it's better now :) 
I just use the number returned from the endpoint to be consistent with the number thats used when splitting into 2 lines